os/board/rtl8720e/src/component/network/ping/ping_test.c

```diff
@@ -3,6 +3,7 @@
 #include "main.h"
 
 #include <lwipconf.h>
+#include <lwip_netconf.h>
```

Comment:
Please revert a file permission to 644.
There are many files which are changed to 755. Please check all of them.